src/ui/theme/components/button.ts

```diff
@@ -17,6 +18,32 @@ const ButtonStyle = {
   },
 };
 
+const sepia = 'rgb(246, 236, 217)';
+const lightGreen = '#468028';
+const lightGrey = '#DADADA';
+const defaultBlack = '#383535';
```

Comment:
These colors should be pulled from the theme wherever possible. I think one thing that would be helpful would be to make a storybook story that renders all the colors in the theme so you can quickly reference them. I'll go ahead and make a PR that does that.